library/std/src/sys/windows/args.rs

```diff
@@ -1,23 +1,30 @@
-#![allow(dead_code)] // runtime init functions not used during testing
+// The Windows command line is just a string
+// https://docs.microsoft.com/en-us/archive/blogs/larryosterman/the-windows-command-line-is-just-a-string
+//
+// This module implements the parsing necessary to turn that string into a list of arguments.
```

Comment:
Maybe
```suggestion
/// The Windows command line is just a string
/// https://docs.microsoft.com/en-us/archive/blogs/larryosterman/the-windows-command-line-is-just-a-string
///
/// This module implements the parsing necessary to turn that string into a list of arguments.
```
?

Reply:
`//!` surely? But yes, that sound sensible.

Reply:
Yeah, `//!`.